tools/shared.py

```diff
@@ -1167,6 +1165,9 @@ def load(self, args=[]):
           declare = re.sub(r'([\w\d]+)\s*=\s*(.+)', r'self.attrs["\1"]=\2;', args[i+1])
           exec declare
 
+      if get_llvm_target() == WASM_TARGET:
```

Comment:
I was about the write the same, yeah, I think it would better to keep backwards compatibility and check that env var